lib/src/judo_table.dart

```diff
@@ -34,6 +34,7 @@ class JudoTable extends StatelessWidget {
     this.inCard = false,
     this.alignment = Alignment.centerLeft,
     this.sortInitially = false,
+    this.tableActions,
```

Comment:
We should initialize this param with a default value. This will negate the need of internal null checks, and prevents potential NPE-s.